.rubocop.yml

```diff
@@ -0,0 +1,82 @@
+require:
+  - rubocop-minitest
+  - rubocop-performance
+
+AllCops:
+  NewCops: disable
+  Exclude:
+    - "bin/**/*"
+    - "Gemfile"
+    - "Gemfile.lock"
+
+Bundler/OrderedGems:
+  Enabled: false
+
+Layout/DotPosition:
+  EnforcedStyle: trailing
+
+Layout/EndOfLine:
+  EnforcedStyle: lf
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Layout/EmptyLinesAroundAccessModifier:
+  EnforcedStyle: only_before
+
+Layout/LineLength:
+  Exclude:
+    - "scripts/generate_fixture.rb"
```

Comment:
I add this exclusion due to [this line](https://github.com/exercism/ruby-representer/blob/master/scripts/generate_fixture.rb#L21). Let me know if you'd prefer I'd rewrite that line.